deluge/ui/Win32IconImagePlugin.py

```diff
@@ -215,6 +215,7 @@ def __repr__(self):
 
 
 class Win32IconImageFile(PIL.ImageFile.ImageFile):
+    # pylint: disable=W0223
```

Comment:
I'll fix this but should be using symbol name `abstract-method` and where possible appended to the line that raises the message.